lib/driver.js

```diff
@@ -122,7 +122,7 @@ class XCUITestDriver extends BaseDriver {
       this.wda = new WebDriverAgent({
         sim: this.sim, //TODO can't be a sim if real device. find a different way to supply logs to webdriveragent
         platformVersion: this.opts.platformVersion,
-        host: this.opts.host,
+        host: this.opts.address,
```

Comment:
Ah! That makes sense. No, I don't think we want to proliferate the number of ways to do things. That seems like a future headache in the making.

Can you, as part of this PR, change `host` to `address` here: https://github.com/appium/appium-xcuitest-driver/blob/master/lib/server.js ? That will make it have `this.opts.address` in the driver itself!